services/backend/server.js

```diff
@@ -79,17 +79,17 @@ router.get("/getEmployeeInfo/:searchValue", keycloak.protect(), async function(
 });
 
 //User endpoints
-router.get("/user/", keycloak.protect(), user.getUser);
-router.get("/user/:id", keycloak.protect(), user.getUserById);
-router.post("/user/", keycloak.protect(), user.createUser);
+router.get("/user/", user.getUser);
+router.get("/user/:id", user.getUserById);
+router.post("/user/", user.createUser);
 
 //Profile endpoints
 router.get("/profile/", keycloak.protect(), profile.getProfile);
 router
   .route("/profile/:id")
-  .get(keycloak.protect(), profile.getPublicProfileById)
-  .post(keycloak.protect(), profile.createProfile)
-  .put(keycloak.protect(), profile.updateProfile);
```

Comment:
Be careful. Our app is now live so stuff like this is risky

Reply:
sorry i keep forgetting to put this back :(